docs/sources/explore/trace-integration.md

```diff
@@ -38,6 +39,7 @@ For information on querying each data source, refer to their documentation:
 - [Jaeger query editor]({{< relref "../datasources/jaeger/#query-the-data-source" >}})
 - [Zipkin query editor]({{< relref "../datasources/zipkin/#query-the-data-source" >}})
 - [Azure Monitor Application Insights query editor]({{< relref "../datasources/azure-monitor/query-editor/#query-application-insights-traces" >}})
+- [ClickHouse query editor](https://clickhouse.com/docs/en/integrations/grafana/query-builder#traces)
```

Comment:
This link doesn't actually work at the moment

Reply:
Apologies yes, there was a mismatch on the timing of these PRs, this page will be available soon. Thank you for the quick review!

Reply:
This page is available now